packages/mdc-notched-outline/mdc-notched-outline.scss

```diff
@@ -68,14 +69,22 @@
     }
 
     &__notch {
+      flex: 0 0 auto;
       width: auto;
+      max-width: 100%;
     }
 
     .mdc-floating-label {
       display: inline-block;
       position: relative;
       top: 17px;
       bottom: auto;
+      max-width: 100%;
+    }
+
+    &--upgraded .mdc-floating-label--float-above {
+      max-width: calc(100% / .75);
+      text-overflow: clip;
```

Comment:
(Discussed in DM) Is it safe to apply this even when not upgraded, to avoid the ellipsis appearing then disappearing once JS runs?